src/topology/algebra/ordered/bounded.lean

```diff
@@ -0,0 +1,48 @@
+/-
+Copyright (c) 2021 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+import topology.algebra.ordered.basic
+
+/-!
+# Convergent sequence is bounded above and below
+
+In this file we prove that a convergent sequence is bounded above and below. We prove it for any
+function that converges along `filter.cofinite`, then specialize to sequences.
+-/
+
+open_locale topological_space
+open set filter
+
+variables {α ι : Type*} [linear_order α] [topological_space α] [order_closed_topology α]
+  {f : ι → α} {l : filter ι} {a : α}
+
+lemma filter.tendsto.exists_ge_eventually_le (h : tendsto f l (𝓝 a)) :
+  ∃ b ≥ a, ∀ᶠ x in l, f x ≤ b :=
+(exists_ge_Iic_mem_nhds a).imp $ λ b hb, ⟨hb.fst, h hb.snd⟩
```

Comment:
We have `eventually_le_of_tendsto_lt` in `topology.algebra.ordered.basic`, which is quite close.  I don't think it would shorten your proof to pass through that, but perhaps they should be cross-referenced, or that lemma could move into this file for unity of theme.

Reply:
Moved.